packages/x-date-pickers/src/AdapterDayjs/AdapterDayjs.ts

```diff
@@ -149,8 +143,6 @@ export class AdapterDayjs implements MuiPickersAdapter<string> {
 
   public lib = 'dayjs';
 
-  public dayjs: Constructor;
```

Comment:
Strictly speaking, this is a breaking change.
Do you think it's worth keeping it (with `withLocale`) even if we never use it?